src/patternfly/utilities/BackgroundColor/examples/BackgroundColor.css

```diff
@@ -0,0 +1,28 @@
+#ws-core-u-background-color-white {
```

Comment:
I don't think we need to set the background on the example container. I would be in favor of just using the utility class on the example like you have it and removing the CSS from the example container. Even when it's a white on white background, or black background with black text. It could be misleading as to what the utility does to present it differently. Maybe in some of the examples, we use a `pf-u-color` utility to also change the text color so it shows up against the background?

Reply:
@mcoker thank you for your feedback -- I initially had tried to just use the utility class on the example, but the background color gets overwritten by the `background-color` of `.ws-preview`, which is set to white, so I had to set it manually on each example container. If you have any ideas for another workaround I'd be happy to make any necessary modifications! what do you think would be the best way to go here?

Reply:
Oh you can just add a div in the example and give it the class of the utility class. Like this

<img width="1236" alt="Screen Shot 2020-10-19 at 1 52 43 PM" src="https://user-images.githubusercontent.com/35148959/96499032-77969280-1212-11eb-96ce-eee724a0a7b9.png">

Reply:
oh excellent - done. thank you!